lib/sqs.go

```diff
@@ -134,10 +146,22 @@ func (sqsQueue *SQSQueue) SendSQSTxnMessage(mempoolTxn *MempoolTx) {
 	}
 }
 
-func makeSubmitPostNotification(mempoolTxn *MempoolTx) *SubmitPostNotification {
+func makePrivateMessageNotification(mempoolTxn *MempoolTx) *PrivateMessageTransaction {
+	metadata := mempoolTxn.Tx.TxnMeta.(*PrivateMessageMetadata)
+	affectedPublicKeys := mempoolTxn.TxMeta.AffectedPublicKeys
+	return &PrivateMessageTransaction{
+		AffectedPublicKeys:             affectedPublicKeys,
+		TransactorPublicKeyBase58Check: mempoolTxn.TxMeta.TransactorPublicKeyBase58Check,
+		RecipientPublicKey:             string(metadata.RecipientPublicKey),
```

Comment:
using string is not a good idea, since these bytes are random bytes - you probably want base58 or at least hex

Reply:
Also, FYI that recipient is the messaging key, so downstream service should be aware of this.